rswag-specs/spec/rswag/specs/configuration_spec.rb

```diff
@@ -3,11 +3,16 @@
 require 'rswag/specs/configuration'
 require 'climate_control'
 
+RspecConfig = Struct.new(
+  :openapi_root, :openapi_specs, :openapi_format, :rswag_dry_run,
+  :openapi_strict_schema_validation, :openapi_all_properties_required, :openapi_no_additional_properties
+)
```

Comment:
You need to pass `keyword_init: true` if you want to use keyword arguments when you initialize with `RspecConfig.new` (see [Keyword Argument](https://ruby-doc.org/core-3.1.0/Struct.html) section in Ruby docs)

```suggestion
RspecConfig = Struct.new(
  :openapi_root, :openapi_specs, :openapi_format, :rswag_dry_run,
  :openapi_strict_schema_validation, :openapi_all_properties_required, :openapi_no_additional_properties,
  keyword_init: true
)
```

Reply:
Looks like you're right; curiously, the tests passed without it though 🤔

Reply:
Yes, that's weird.